File: cases/taint/php/expression-positive/expression_positive.php
<?php
function dfb_source(): int { // DFB-SOURCE: expression-input
    return 4;
}

function dfb_sink(int $value): void {} // DFB-SINK: expression-sink

function run(): void {
    $value = dfb_source();
    $computed = ($value * 3) + 7; // DFB-WITNESS: expression-computed
    dfb_sink($computed);
}

File: cases/taint/rust/expression-positive/expression_positive.rs
fn dfb_source() -> i32 { // DFB-SOURCE: expression-input
    1
}

fn dfb_sink(value: i32) {} // DFB-SINK: expression-sink

fn run() {
    let value = dfb_source();
    let computed = (value * 3) + 7; // DFB-WITNESS: expression-computed
    dfb_sink(computed);
}
